src/SectionsNavigation.Uno/SectionsNavigation.Uno.csproj

```diff
@@ -3,7 +3,7 @@
 	<PropertyGroup>
 		<!-- Change the TargetFrameworks depending on which platform you are building on. This avoids errors as it is impossible to build UAP on OSX (MacOS) -->
 		<TargetFrameworks Condition="'$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;</TargetFrameworks>
-		<TargetFrameworks Condition="'!$([MSBuild]::IsOsPlatform(OSX))'">netstandard2.0;xamarinios10;monoandroid11.0;monoandroid12.0;uap10.0.19041</TargetFrameworks>
```

Comment:
I think we should probably mark this as a breaking change (removing support for android 11).

Reply:
I added this to the list of breaking changes and marked the commit as breaking with the ! signifier